src/SourceLocator/Type/Composer/Factory/MakeLocatorForComposerJsonAndInstalledJson.php

```diff
@@ -180,9 +185,9 @@ private function packagePrefixPath(string $trimmedInstallationPath, array $packa
      *
      * @return array<string, array<int, string>>
      */
-    private function prefixWithPackagePath(array $paths, string $trimmedInstallationPath, array $package): array
+    private function prefixWithPackagePath(array $paths, string $trimmedInstallationPath, array $package, string $vendorDir = 'vendor'): array
```

Comment:
`$vendorDir` no longer needs a default, since it is always declared and passed in explicitly